cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/hdfs/HdfsDelegationTokenRefresherConfig.java

```diff
@@ -0,0 +1,55 @@
+package com.linkedin.openhouse.cluster.storage.hdfs;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ * Configuration class to conditionally enable/disable creation of {@link
+ * HdfsDelegationTokenRefresher} bean and enable delegation token refresh scheduling.
+ */
+@Slf4j
+@Configuration
+@EnableScheduling
+public class HdfsDelegationTokenRefresherConfig {
```

Comment:
I believe this is the existing class and rename is done. With some refactoring, the functionality remains the same, right?

Reply:
yes!